src/sage/combinat/tuple.py

```diff
@@ -90,6 +90,88 @@ def __repr__(self):
         """
         return "Tuples of %s of length %s" % (self.S, self.k)
 
+    def unrank(self, i):
+        """
+        Return the `i`-th element of the set of Tuples.
+
+        INPUT:
+
+        - ``i`` -- integer between `0` and `n-1` where `n` is the cardinality
+          of this set.
+
+        EXAMPLES::
+
+            sage: T = Tuples(range(7), 6)
+            sage: T[73451]
+            (0, 0, 1, 4, 2, 4)
+
+        TESTS:
+
+        Verify that `unrank` is giving the correct result. ::
```

Comment:
```suggestion
        Verify that :meth:`unrank` is giving the correct result::
```
and similar changes.